src/common/Smi.Common/Options/GlobalOptions.cs

```diff
@@ -486,7 +486,15 @@ public class DicomAnonymiserOptions : IOptions
         public string? RoutingKeySuccess { get; set; }
         public string? RoutingKeyFailure { get; set; }
         public bool FailIfSourceWriteable { get; set; } = true;
-
+        public string? VirtualEnvPath { get; set; }
+        public string? ShellScriptPath { get; set; }
+        public string? DicomPixelAnonPath { get; set; }
+        public string? SmiServicesPath { get; set; }
```

Comment:
I don't think we need this, as we shouldn't need to pass `SMI_ROOT` to the Processes.

Reply:
Action: (da) SmiServicesPath removed from CreateDICOMProcess and CreateCTPProcess. 
Action: (da) Remove SmiServicesPath from CreateSRProcess after reviewing the SR script.